samples/photo_album/src/index.js

```diff
@@ -1,21 +1,21 @@
 import React from 'react';
-import {render} from 'react-dom';
-import {createStore, combineReducers} from 'redux';
-import {Provider} from 'react-redux';
+import { render } from 'react-dom';
+import { createStore, combineReducers } from 'redux';
+import { Provider } from 'react-redux';
 import AppContainer from './components/App';
 import PhotosListReducer from './reducers/PhotosListReducer';
 import UploadedPhotosReducer from './reducers/UploadedPhotosReducer';
 
 const rootReducer = combineReducers({
     photos: PhotosListReducer,
-    uploadedPhotos: UploadedPhotosReducer
+    uploadedPhotos: UploadedPhotosReducer,
 });
 
 const store = createStore(rootReducer);
 
 render(
     <Provider store={store}>
-        <AppContainer cloudName='dvxpdqpjj' uploadPreset='hvs8orho'/>
+        <AppContainer cloudName="danashalev" uploadPreset="hvs8orho" />
```

Comment:
This shouldn't have your credentials. Read cloud name + upload preset from a config file provided by the user (you can create a sample config file with placeholders).